verilog/tools/ls/autoexpand.cc

```diff
@@ -1052,29 +1077,40 @@ std::optional<AutoExpander::Expansion> AutoExpander::ExpandAutoinst(
   }
   const Module &inst_module = modules_.at(type_id);
 
+  // Find an AUTO_TEMPLATE that matches this instance
+  const verible::TokenInfo *instance_name_token =
+      GetModuleInstanceNameTokenInfoFromGateInstance(instance);
+  if (!instance_name_token) {
+    LOG(ERROR) << "AUTOINST: Instance with no name, aborting";
+    return {};
```

Comment:
In a separate PR, but I think we should change all the `std::nullopt` returns from `{}` to actual `std::nullopt` .
Reading the code it can be confusing as it could be mistaken for an empty struct initialization of the std::optional value. Being explicit is nicer there.

Reply:
https://github.com/chipsalliance/verible/pull/1841